Form/SpecTypeExtension.php

```diff
@@ -18,7 +17,7 @@ public function configureOptions(OptionsResolver $resolver)
         $this->setDefaultOptions($resolver);
     }
 
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    public function setDefaultOptions(OptionsResolver $resolver)
```

Comment:
Nice catch 👍 

Aren't we also supposed to define the `configureOptions` method instead of `setDefaultOptions`? (cf: https://symfony.com/doc/2.8/forms.html#creating-form-classes)

Reply:
Note to self: it's already defined in the file (and it calls `setDefaultOptions`).

It's probably done this way to ensure BC for Symfony <2.8. I'll make sure that it's really necessary to maintain compatibility for <2.8 and if not, I'll cleanup the code.